e2e/multisig-migration.test.ts

```diff
@@ -107,6 +114,71 @@ describe("Should change LBC owner to the multisig.ts", function () {
     );
     await expect(newLbc.version()).to.eventually.be.equal("1.3.0");
   });
+
+  it("Should change the owner using Tenderly", async () => {
+    await checkForkedNetwork();
+    const deploymentNetwork = hre.network.name;
+    const multisigAddress = multsigInfo[deploymentNetwork].address;
+    if (!multisigAddress || multisigAddress === "")
+      throw new Error("Multisig address not found for current network");
+
+    // Get the current owner of LiquidityBridgeContract
+    const deploymentData = read()[deploymentNetwork];
+    const proxyAddress = deploymentData.LiquidityBridgeContract.address;
+    if (!proxyAddress)
+      throw new Error(
+        `LiquidityBridgeContract address not found on network ${deploymentNetwork}`
+      );
+
+    const contract = await ethers.getContractAt(
+      "LiquidityBridgeContractV2",
+      proxyAddress
+    );
+    const currentOwner = await contract.owner();
+    console.info(`Current owner: ${currentOwner}`);
+
+    // Impersonate the current owner account
+    await helpers.impersonateAccount(currentOwner);
+    const impersonatedSigner = await ethers.getSigner(currentOwner);
+
+    // Set balance for the impersonated account
+    const desiredBalance = ethers.toQuantity(ethers.parseEther("100"));
+    await ethers.provider.send("hardhat_setBalance", [
+      impersonatedSigner.address,
+      desiredBalance,
+    ]);
+
+    console.info(`Impersonated account: ${impersonatedSigner.address}`);
+
+    // Call changeMultisigOwner
+    await expect(
+      changeMultisigOwner(
+        multisigAddress,
+        deploymentNetwork,
+        impersonatedSigner
+      )
+    ).to.not.be.reverted;
+
+    const newOwner = await contract.owner();
+    console.info(`New owner: ${newOwner}`);
+
+    // Verify ownership change
+    expect(newOwner.toLowerCase()).to.equal(multisigAddress.toLowerCase());
```

Comment:
you are verifying the owner of the contract, which is correct, but you also need to verify the owner of the proxy admin (they both change)